test/src/system/pasting_test.coffee

```diff
@@ -85,6 +85,13 @@ testGroup "Pasting", template: "editor_empty", ->
       assert.textAttributes([0, 7], href: "http://example.com")
       expectDocument "Example\n"
 
+  test "paste JavaScript URL", (expectDocument) ->
+   pasteData =
+     "URL": "javascript:alert('XSS')\n"
+   pasteContent pasteData, ->
+     assert.textAttributes([0, 23], {})
+     expectDocument "javascript:alert('XSS') \n"
```

Comment:
Is the space before the `\n` expected?

Reply:
When I added `\n`, the test failed. The diff showed that a space was missing, as soon as I added a space before `\n` the test passed.

Reply:
Since I am removing the `\n`, I can remove the space before `\n`